internal/endtoend/testdata/func_args/go/query.sql.go

```diff
@@ -11,9 +11,9 @@ const makeIntervalDays = `-- name: MakeIntervalDays :one
 SELECT make_interval(days => $1::int)
 `
 
-func (q *Queries) MakeIntervalDays(ctx context.Context, dollar_1 int32) (interface{}, error) {
+func (q *Queries) MakeIntervalDays(ctx context.Context, dollar_1 int32) (int64, error) {
```

Comment:
`make_interval` now returns an actual type instead of `interface{}`. This happened because we can now resolve function calls with defaults and named parameters.